lib/read-stream.js

```diff
@@ -246,20 +246,29 @@ ReadStream.prototype = {
       return void this.once( 'open', () => { this._read() })
     }
 
-    if( this.ranges.length === 0 ) {
-      this._verifyRange()
-      return void this.push( null )
-    }
-
-    if( this.currentRange == null || this.currentRange.offset === this.currentRange.length ) {
-      this._verifyRange()
-      this.currentRange = this.ranges.shift()
-      this.rangesRead++
-      debug( 'read:range %O', this.currentRange )
```

Comment:
Should this `debug` statement be preserved, or doesn't it matter?

Reply:
Good catch @lurch , `debug` statement added back.